### PR TITLE
remove xml and readme fix

### PR DESCRIPTION
**Pull request by trs-80**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,10 +67,10 @@ version 0.011
 
     # $link is an arrayfef of the all the playlist urls
 
-    foreach my $for_tracks (@{$link}) {
+    foreach my $playlist (@{$link}) {
```
Comment (oalders) [inline README.md:70]: No need to edit the `README` as this will be done automatically a part of the release process.

**Review by oalders — APPROVED** [review]
Looks great! You may want to include a Changes entry for this as well.

**Comment by trs-80** [comment]
@oalders would appreciate if you could do the release to CPAN now that these changes have been merged to master. I attempted to do it myself but had some issues. Also I appear to have borked the automated version update some how. Unsure if it an issue with local files or something in the public commits.

**Comment by oalders** [comment]
I'll get a new release out. The issue is that at the version number was decreased manually in fb54743e. This is handled by `dzil` and it's now a bit confused.

**Comment by oalders** [comment]
`V=0.012 dzil release` manually overrides the automated version bump. New release on the way to CPAN.

**Comment by oalders** [comment]
The version has been bumped for the next release already, so there's no need to change it moving forward.

**Pull request merged**